src/types.rs

```diff
@@ -70,6 +70,7 @@ impl Data {
         ctx: &serenity::Context,
     ) -> anyhow::Result<()> {
         let action = self.config.get_response(name.to_string());
+        println!("{:?}", action.kind);
```

Comment:
Is this a leftover debug line, or do we want the bot to actually log this stuff?